### DISCUSSION TITLE
How to listen to a family provider for any families instances updates?

### DISCUSSION THREAD

**Discussion by Maus3rSR** [rrousselGit/riverpod#2969] [Q&A] [answered]
Hello,

Iwould like to listen to any of families instance update like this: 
`ref.listen(retrieveWorkDetailUseCaseProvider, (previous, next) {});` where next will contain the family context and the result of it.

There is some workaround?

**Comment by rrousselGit** [comment, marked as answer]
That is not possible with ref.listen, as it wouldn't know what to listen.

You can use ProviderObserver though, and listen for updates in there.

**Reply by Maus3rSR** [reply]
Thank you. I will do it through the ProviderObserver.
